yarn.lock

```diff
@@ -1892,17 +1892,6 @@
     path-browserify "^1.0.0"
     url "^0.11.0"
 
-"@sentry/core@5.30.0":
```

Comment:
How come so much sentry stuff got removed?

Reply:
i have no idea, might be all concentrated into this line now? https://github.com/ethereum-optimism/optimism/pull/722/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR1942 lmk if you think i should retry the package change though!

Reply:
just reran yarn and it got added back in - not sure what happened but will push with the next commit!